<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveMeasurementsWindow</class>
 <widget class="QMainWindow" name="saveMeasurementsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="accPlot" native="true"/>
        </item>
        <item>
         <widget class="QCustomPlot" name="gyroPlot" native="true"/>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QPushButton" name="startButton">
                <property name="text">
                 <string>start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cutFirst20Button">
                <property name="text">
                 <string>cut first 20 samples</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QPushButton" name="stopButton">
                <property name="text">
                 <string>stop</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cutLast20Button">
                <property name="text">
                 <string>cut last 20 samples</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="okGraphButton">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="terminalButton">
            <property name="text">
             <string>terminal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quitSaveWindowButton">
            <property name="text">
             <string>quit</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>skater:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="skaterBox"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>trick:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="tricksBox"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="onSaveAsCSVflie">
              <property name="text">
               <string>save as .csv flie</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPlainTextEdit" name="CSVText">
              <property name="plainText">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="backButton">
            <property name="text">
             <string>back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quitSaveWindowButton</sender>
   <signal>clicked()</signal>
   <receiver>saveMeasurementsWindow</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
